platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/version/AGWorkflowVersionTable.java

```diff
@@ -48,6 +50,12 @@ public void customizeAuthorizer(EmfAuthorizer<AGWorkflowVersion, AGWorkflow> aut
 	public void customizeManagementConfiguraton(EmfManagementConfiguration<AGWorkflowVersion> configuration) {
 
 		configuration.addOrderBy(AGWorkflowVersion.ID, OrderDirection.DESCENDING);
+		configuration.setRowCustomizer(row -> {
+			row.addCssClass(WorkflowCssClasses.WORKFLOW_VERSION_ROW);
+			if (row.getDataRow().isCurrentVersion()) {
+				row.addCssClass(DomCssPseudoClasses.ACTIVE);
+			}
+		});
```

Comment:
I think the content of the curly braces should be extracted to a private method, e.g. `void customizeRow(AGWorkflowVersion)`
Then:
`configuration.setRowCustomizer(this::customizeRow);`